Lombiq.HelpfulExtensions/Views/GoogleTag.cshtml

```diff
@@ -15,7 +15,7 @@
 {
     GoogleAnalyticsContentSecurityPolicyProvider.EnableForCurrentRequest(Context);
 
-    var viewModel = Model.ViewModel as GoogleTagViewModel ?? new GoogleTagViewModel
+    var viewModel = Model.GetType().GetProperty("ViewModel") as GoogleTagViewModel ?? new GoogleTagViewModel
```

Comment:
I tested this locally by temporarily deleting temporarily the `if` statement. It no longer crashes.